fury/actor/actor.py

```diff
@@ -0,0 +1,54 @@
+import fury.primitive as fp
```

Comment:
not sure actor should be under actor folder. let's keep it simple and create submodule later

Reply:
@m-agour  and @skoudoro The broader picture on this is, it should have `actor` directory and actors should be divided in files such as sphere and line should be part of `base.py` and odf and slicer should be part of `medical.py` segregating the actors based on buckets or types.